app/ollama.iss

```diff
@@ -91,6 +91,14 @@ Source: "..\ollama.exe"; DestDir: "{app}"; Flags: ignoreversion 64bit
 Source: "..\dist\windeps\*.dll"; DestDir: "{app}"; Flags: ignoreversion 64bit
 Source: "..\dist\ollama_welcome.ps1"; DestDir: "{app}"; Flags: ignoreversion
 Source: ".\assets\app.ico"; DestDir: "{app}"; Flags: ignoreversion
+; Assumes v5.7, may need adjustments for v6
+#if GetEnv("HIP_PATH") != ""
```

Comment:
This may be an area for improvement in the future where if we write some custom pascal script code we could detect presence of ROCm on the system and skip these files.  If we were to get super fancy, the ROCm part could be a separate download.  That said, it didn't bloat the installer as much as I had feared, so this felt like a reasonable compromise for now.  There's also a potential for us to generate a custom build of the tensor library to attempt to support additional GPUs.  For the foreseeable future, I anticipate we'll be shipping more often that ROCm on windows.

Reply:
Do we want to always extract these anyways?

Reply:
Right now, it's always installing ROCm on windows, even if the system doesn't have a Radeon card, there's no conditional logic.  It's debatable if we should "live off the land" if ROCm is present in the future, but that's not in this PR.